backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -12,13 +12,21 @@ Because all taken backup remains until expired, also check the time to live (TTL
 
 
 You can back up applications by creating a `Backup` custom resource (CR). For more information, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
+The following are the different backup types for a `Backup` CR:
 
 ifdef::openshift-rosa,openshift-rosa-hcp[]
 The `Backup` CR creates backup files for Kubernetes resources and internal images on S3 object storage.
 endif::openshift-rosa,openshift-rosa-hcp[]
 
 ifndef::openshift-rosa,openshift-rosa-hcp[]
 * The `Backup` CR creates backup files for Kubernetes resources and internal images on S3 object storage.
+
+* In an on-cluster backup, you back up the persistent volume data within the cluster. This type of backup is fast, and you can execute it frequently; however, the data is vulnerable to cluster or storage outages. You can use Container Storage Interface (CSI) volume snapshots for an on-cluster backup.
```

Comment:
@weshayutin @apurvabhide17 I'm thinking change it to something like this 
```
if you use velero's snapshot feature to backup volumes in that case only snapshot related information is stored in s3 bucket along with the openshift object data. If undelying storage is part of same cluster infra then data might be loss in case of disaster.  Same for bucket if bucket is part of the same cluster then all backup data will be lost in case of disaster.
```